app/views/hearings/show.html.haml

```diff
@@ -71,7 +70,7 @@
   = govuk_detail('View raw data') do
     %h3='Hearings'
```

Comment:
All content string be added to locale file.

Reply:
done